pkg/crc/machine/vsock.go

```diff
@@ -92,10 +92,15 @@ func vsockPorts(preset crcPreset.Preset, ingressHTTPPort, ingressHTTPSPort uint)
 		socketProtocol = types.NPIPE
 		socketLocal = constants.DefaultPodmanNamedPipe
 	}
+
+	// API, Cockpit and Internal SSH are all bound to a private address, usually 127.0.0.1
+	// On Windows, where WSL2 is active, it's bound to the Windows machine's address within the private WSL2 network
+	privateIP := VsockPrivateAddress()
```

Comment:
this should also be behind the config option, now on windows it is always binding the port forwards to the WSL vswitch ip address

Reply:
this is handled now in https://github.com/crc-org/crc/pull/3337/commits/e0ef98d82a0f21a65ee8b8adb1bd320ac621944e